src/main/msp/msp.c

```diff
@@ -1279,11 +1280,43 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, flight3DConfig()->deadband3d_throttle);
         break;
 
-    case MSP_SENSOR_ALIGNMENT:
-        sbufWriteU8(dst, gyroConfig()->gyro_align);
-        sbufWriteU8(dst, accelerometerConfig()->acc_align);
+    case MSP_SENSOR_ALIGNMENT: {
+        uint8_t gyroAlignment = gyroDeviceConfig(0)->align;
```

Comment:
I think that the initialization is not needed, it is done in the if else later

Reply:
Yes you're right. Leftover from an earlier method I first tested. I'll fix in a bit.